From: Mauro Carvalho Chehab <mchehab@redhat.com>
Subject: edac/Kconfig: edac_mce can't be module
References: fate#311968
Git-commit: 963c5ba35984c87963480031d1d7e2e556256ad7
Patch-mainline: v2.6.35-rc2


Signed-off-by: Thomas Renninger <trenn@suse.de>

Since mcelog is bool, edac_mce glue should also be bool, or otherwise
will not work.

Signed-off-by: Mauro Carvalho Chehab <mchehab@redhat.com>

 drivers/edac/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/edac/Kconfig b/drivers/edac/Kconfig
index 5b7fbc5..aedef79 100644
--- a/drivers/edac/Kconfig
+++ b/drivers/edac/Kconfig
@@ -70,7 +70,7 @@ config EDAC_MM_EDAC
 	  replaced.  If unsure, select 'Y'.
 
 config EDAC_MCE
-	tristate
+	bool
 
 config EDAC_AMD64
 	tristate "AMD64 (Opteron, Athlon64) K8, F10h, F11h"
